app/src/sharedTest/java/org/oppia/app/walkthrough/WalkthroughActivityTest.kt

```diff
@@ -12,14 +12,14 @@ import org.junit.runner.RunWith
 import org.oppia.app.R
 import org.oppia.app.utility.ProgressMatcher.Companion.withProgress
 
-/** Tests for [WalkthroughFragment]. */
+/** Tests for [WalkthroughActivity]. */
 @RunWith(AndroidJUnit4::class)
-class OnboardingFragmentTest {
+class WalkthroughActivityTest {
```

Comment:
Add one test case for default progress too. Also, the `back_button` ImageView is working correctly but we expect that same behaviour from default Android-OS back button, so write test cases for that too.

Reply:
done